> OK, let me know how `make -C Documentation update-cmdref` goes on Linux. If it's still failing, I'll just run it here on top of your changes and pass you the diff. You're not expected to spend hours on debugging/fixing that command when contributing a PR :).

thanks! sad, it's still failing...

```
[root@localhost cilium]# make -C Documentation update-cmdref
make: 进入目录“/root/cyclinder/cilium/Documentation”
:
# Pre-pull FROM docker image due to Buildkit sometimes failing to pull them.
grep -m 1 "^FROM " Dockerfile | tr -d '\r' | cut -d ' ' -f2 | xargs -n1 docker pull
3.10.4-alpine3.15: Pulling from library/python
Digest: sha256:11a95bbf024fc1d116e83d13669e0c16686e271c7c78d5b1436db37672167e5f
Status: Image is up to date for python:3.10.4-alpine3.15
docker.io/library/python:3.10.4-alpine3.15
tar c requirements.txt Dockerfile | docker build  --target docs-builder --tag cilium/docs-builder -
Sending build context to Docker daemon  10.24kB
Step 1/10 : FROM docker.io/library/python:3.10.4-alpine3.15 AS docs-base
 ---> abd9fc366a36
Step 2/10 : LABEL maintainer="maintainer@cilium.io"
 ---> Using cache
 ---> c7a2b8661cfc
Step 3/10 : RUN apk add --no-cache --virtual --update     aspell-en     nodejs     npm     bash     ca-certificates     enchant2     enchant2-dev     git     libc6-compat     py-pip     python3     py3-sphinx     gcc     musl-dev     && true
 ---> Using cache
 ---> 68bb57dac1e3
Step 4/10 : FROM docs-base AS docs-builder
 ---> 68bb57dac1e3
Step 5/10 : ADD ./requirements.txt /tmp/requirements.txt
 ---> Using cache
 ---> e0914f07ef92
Step 6/10 : RUN pip install -r /tmp/requirements.txt
 ---> Using cache
 ---> 174f630132dc
Step 7/10 : ENV HOME=/tmp
 ---> Using cache
 ---> 8558671076fa
Step 8/10 : ENV READTHEDOCS_VERSION=$READTHEDOCS_VERSION
 ---> Using cache
 ---> ab0d1eea7b90
Step 9/10 : ENV MAKE_GIT_REPO_SAFE=1
 ---> Using cache
 ---> fae7eb628f17
Step 10/10 : RUN install -m 0777 -d /usr/local/lib/python3.10/site-packages/versionwarning/_static/data
 ---> Using cache
 ---> e0bbdd9965e7
Successfully built e0bbdd9965e7
Successfully tagged cilium/docs-builder:latest
make -C ../ build
make[1]: 进入目录“/root/cyclinder/cilium”
pkg/datapath/loader/check-sources.sh
make[2]: 进入目录“/root/cyclinder/cilium/proxylib”
CGO_ENABLED=1  go build  -mod=vendor -ldflags ' -X "github.com/cilium/cilium/pkg/version.ciliumVersion=1.13.90 84f6cbc %aI" -s -w -X "github.com/cilium/cilium/pkg/envoy.RequiredEnvoyVersionSHA=ff9a74a473dd5c1f6bfd7ce10938dab648f70c09" -X "github.com/cilium/cilium/pkg/datapath/loader.DatapathSHA256=d15d02fe7d4a7aa0f0aee009cbb7401d7c035ff0045d20542e57d95eb0d4ed06" -extldflags -Wl,-soname,libcilium.so.1' -tags=osusergo  -o libcilium.so.1 -buildmode=c-shared
ln -sf libcilium.so.1 libcilium.so || cp libcilium.so.1 libcilium.so
make[2]: 离开目录“/root/cyclinder/cilium/proxylib”
make[2]: 进入目录“/root/cyclinder/cilium/envoy”
  SKIP  envoy/
cilium-envoy has been moved to github.com/cilium/proxy
make[2]: 离开目录“/root/cyclinder/cilium/envoy”
make[2]: 进入目录“/root/cyclinder/cilium/bpf”
set -e; \
if [ "" != "" ]; then \
	 :  " [-DALLOW_ICMP_FRAG_NEEDED=1 ]"; clang -DALLOW_ICMP_FRAG_NEEDED=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DSKIP_DEBUG=1 ]"; clang -DSKIP_DEBUG=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DHAVE_LPM_TRIE_MAP_TYPE=1 ]"; clang -DHAVE_LPM_TRIE_MAP_TYPE=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DHAVE_LRU_HASH_MAP_TYPE=1 ]"; clang -DHAVE_LRU_HASH_MAP_TYPE=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENABLE_IPV4=1 ]"; clang -DENABLE_IPV4=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENABLE_IPV6=1 ]"; clang -DENABLE_IPV6=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENABLE_IPV4=1 -DENABLE_IPV6=1 -DPOLICY_VERDICT_NOTIFY=1 -DENABLE_IPSEC=1 -DIP_POOLS=1 ]"; clang -DENABLE_IPV4=1 -DENABLE_IPV6=1 -DPOLICY_VERDICT_NOTIFY=1 -DENABLE_IPSEC=1 -DIP_POOLS=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENABLE_IPV4=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 ]"; clang -DENABLE_IPV4=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENABLE_IPV4=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DENABLE_IPSEC=1 -DIP_POOLS=1 ]"; clang -DENABLE_IPV4=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DENABLE_IPSEC=1 -DIP_POOLS=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENABLE_IPV4=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DHAVE_LPM_TRIE_MAP_TYPE=1 ]"; clang -DENABLE_IPV4=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DHAVE_LPM_TRIE_MAP_TYPE=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENABLE_IPV4=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DHAVE_LPM_TRIE_MAP_TYPE=1 -DENABLE_EGRESS_GATEWAY=1 ]"; clang -DENABLE_IPV4=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DHAVE_LPM_TRIE_MAP_TYPE=1 -DENABLE_EGRESS_GATEWAY=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENABLE_IPV4=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DHAVE_LPM_TRIE_MAP_TYPE=1 -DHAVE_LRU_HASH_MAP_TYPE=1 ]"; clang -DENABLE_IPV4=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DHAVE_LPM_TRIE_MAP_TYPE=1 -DHAVE_LRU_HASH_MAP_TYPE=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENABLE_IPV4=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DENABLE_IPV4_FRAGMENTS=1 ]"; clang -DENABLE_IPV4=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DENABLE_IPV4_FRAGMENTS=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENABLE_IPV6=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 ]"; clang -DENABLE_IPV6=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENABLE_IPV6=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DENABLE_IPSEC=1 -DIP_POOLS=1 ]"; clang -DENABLE_IPV6=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DENABLE_IPSEC=1 -DIP_POOLS=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENABLE_IPV6=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DHAVE_LPM_TRIE_MAP_TYPE=1 ]"; clang -DENABLE_IPV6=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DHAVE_LPM_TRIE_MAP_TYPE=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENABLE_IPV6=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DHAVE_LPM_TRIE_MAP_TYPE=1 -DHAVE_LRU_HASH_MAP_TYPE=1 ]"; clang -DENABLE_IPV6=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DHAVE_LPM_TRIE_MAP_TYPE=1 -DHAVE_LRU_HASH_MAP_TYPE=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENABLE_IPV6=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DENABLE_IPV4=1 ]"; clang -DENABLE_IPV6=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DENABLE_IPV4=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENABLE_IPV6=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DENABLE_IPV4=1 -DENABLE_ROUTING=1 ]"; clang -DENABLE_IPV6=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DENABLE_IPV4=1 -DENABLE_ROUTING=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENABLE_IPV6=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DENABLE_IPV4=1 -DENABLE_IPSEC=1 -DIP_POOLS=1 -DENABLE_ENCAP_HOST_REMAP=1 ]"; clang -DENABLE_IPV6=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DENABLE_IPV4=1 -DENABLE_IPSEC=1 -DIP_POOLS=1 -DENABLE_ENCAP_HOST_REMAP=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENABLE_IPV6=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DENABLE_IPV4=1 -DENABLE_IPSEC=1 -DIP_POOLS=1 -DENABLE_ENCAP_HOST_REMAP=1 -DENABLE_L7_LB=1 ]"; clang -DENABLE_IPV6=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DENABLE_IPV4=1 -DENABLE_IPSEC=1 -DIP_POOLS=1 -DENABLE_ENCAP_HOST_REMAP=1 -DENABLE_L7_LB=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENABLE_IPV4=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DENABLE_IPV6=1 -DHAVE_LPM_TRIE_MAP_TYPE=1 -DHAVE_LRU_HASH_MAP_TYPE=1 ]"; clang -DENABLE_IPV4=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DENABLE_IPV6=1 -DHAVE_LPM_TRIE_MAP_TYPE=1 -DHAVE_LRU_HASH_MAP_TYPE=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENABLE_IPV4=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DENABLE_IPV6=1 -DHAVE_LPM_TRIE_MAP_TYPE=1 -DHAVE_LRU_HASH_MAP_TYPE=1 -DENABLE_TPROXY=1 ]"; clang -DENABLE_IPV4=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DENABLE_IPV6=1 -DHAVE_LPM_TRIE_MAP_TYPE=1 -DHAVE_LRU_HASH_MAP_TYPE=1 -DENABLE_TPROXY=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENABLE_IPV4=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DENABLE_IPV6=1 -DHAVE_LPM_TRIE_MAP_TYPE=1 -DHAVE_LRU_HASH_MAP_TYPE=1 -DENABLE_TPROXY=1 -DENABLE_L7_LB=1 ]"; clang -DENABLE_IPV4=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DENABLE_IPV6=1 -DHAVE_LPM_TRIE_MAP_TYPE=1 -DHAVE_LRU_HASH_MAP_TYPE=1 -DENABLE_TPROXY=1 -DENABLE_L7_LB=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DENABLE_IPV4=1 -DENABLE_IPV6=1 -DPOLICY_VERDICT_NOTIFY=1 ]"; clang -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DENABLE_IPV4=1 -DENABLE_IPV6=1 -DPOLICY_VERDICT_NOTIFY=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DENABLE_IPV4=1 -DENABLE_IPV6=1 -DPOLICY_VERDICT_NOTIFY=1 -DENABLE_NAT_46X64=1 -DENABLE_NAT_46X64_GATEWAY=1 ]"; clang -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DENABLE_IPV4=1 -DENABLE_IPV6=1 -DPOLICY_VERDICT_NOTIFY=1 -DENABLE_NAT_46X64=1 -DENABLE_NAT_46X64_GATEWAY=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DENABLE_NODEPORT=1 -DENABLE_IPV4=1 -DENABLE_IPV6=1 -DPOLICY_VERDICT_NOTIFY=1 ]"; clang -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DENABLE_NODEPORT=1 -DENABLE_IPV4=1 -DENABLE_IPV6=1 -DPOLICY_VERDICT_NOTIFY=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DENABLE_NODEPORT=1 -DENABLE_DSR=1 -DENABLE_IPV4=1 -DENABLE_IPV6=1 -DPOLICY_VERDICT_NOTIFY=1 ]"; clang -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DENABLE_NODEPORT=1 -DENABLE_DSR=1 -DENABLE_IPV4=1 -DENABLE_IPV6=1 -DPOLICY_VERDICT_NOTIFY=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENABLE_IPV4=1 -DENABLE_IPV6=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DUSE_BPF_PROG_FOR_INGRESS_POLICY=1 ]"; clang -DENABLE_IPV4=1 -DENABLE_IPV6=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DPOLICY_VERDICT_NOTIFY=1 -DUSE_BPF_PROG_FOR_INGRESS_POLICY=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENABLE_IPV4=1 -DENABLE_IPV6=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DHAVE_LPM_TRIE_MAP_TYPE=1 -DHAVE_LRU_HASH_MAP_TYPE=1 -DENABLE_TPROXY=1 -DENABLE_HOST_ROUTING=1 ]"; clang -DENABLE_IPV4=1 -DENABLE_IPV6=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DHAVE_LPM_TRIE_MAP_TYPE=1 -DHAVE_LRU_HASH_MAP_TYPE=1 -DENABLE_TPROXY=1 -DENABLE_HOST_ROUTING=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENABLE_IPV4=1 -DENABLE_IPV6=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DHAVE_LPM_TRIE_MAP_TYPE=1 -DHAVE_LRU_HASH_MAP_TYPE=1 -DENABLE_TPROXY=1 -DENABLE_HOST_ROUTING=1 -DENABLE_SKIP_FIB=1 ]"; clang -DENABLE_IPV4=1 -DENABLE_IPV6=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DHAVE_LPM_TRIE_MAP_TYPE=1 -DHAVE_LRU_HASH_MAP_TYPE=1 -DENABLE_TPROXY=1 -DENABLE_HOST_ROUTING=1 -DENABLE_SKIP_FIB=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENABLE_IPV4=1 -DENABLE_IPV6=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DHAVE_LPM_TRIE_MAP_TYPE=1 -DHAVE_LRU_HASH_MAP_TYPE=1 -DENABLE_TPROXY=1 -DENABLE_HOST_ROUTING=1 -DENABLE_SKIP_FIB=1 -DENABLE_WIREGUARD=1 ]"; clang -DENABLE_IPV4=1 -DENABLE_IPV6=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DHAVE_LPM_TRIE_MAP_TYPE=1 -DHAVE_LRU_HASH_MAP_TYPE=1 -DENABLE_TPROXY=1 -DENABLE_HOST_ROUTING=1 -DENABLE_SKIP_FIB=1 -DENABLE_WIREGUARD=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;   :  " [-DENABLE_IPV4=1 -DENABLE_IPV6=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DHAVE_LPM_TRIE_MAP_TYPE=1 -DHAVE_LRU_HASH_MAP_TYPE=1 -DENABLE_TPROXY=1 -DENABLE_HOST_ROUTING=1 -DENABLE_SKIP_FIB=1 -DENABLE_WIREGUARD=1 -DENABLE_ICMP_RULE=1 -DENABLE_SCTP=1 ]"; clang -DENABLE_IPV4=1 -DENABLE_IPV6=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -DHAVE_LPM_TRIE_MAP_TYPE=1 -DHAVE_LRU_HASH_MAP_TYPE=1 -DENABLE_TPROXY=1 -DENABLE_HOST_ROUTING=1 -DENABLE_SKIP_FIB=1 -DENABLE_WIREGUARD=1 -DENABLE_ICMP_RULE=1 -DENABLE_SCTP=1  -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll; llc -march=bpf -mcpu=v2 -o /dev/null bpf_lxc.ll;  \
fi
clang -DSKIP_DEBUG=1 -DENABLE_IPV4=1 -DENABLE_IPV6=1 -DENABLE_ROUTING=1 -DPOLICY_VERDICT_NOTIFY=1 -DALLOW_ICMP_FRAG_NEEDED=1 -DENABLE_IDENTITY_MARK=1 -DMONITOR_AGGREGATION=3 -DCT_REPORT_FLAGS=0x0002 -DENABLE_HOST_FIREWALL=1 -DENABLE_ICMP_RULE=1 -DENABLE_CUSTOM_CALLS=1 -DENABLE_SRV6=1 -DENABLE_L7_LB=1 -DHAVE_LPM_TRIE_MAP_TYPE=1 -DHAVE_LRU_HASH_MAP_TYPE=1 -DENABLE_MASQUERADE=1 -DENABLE_SRC_RANGE_CHECK=1 -DENABLE_NODEPORT=1 -DENABLE_NODEPORT_ACCELERATION=1 -DENABLE_SESSION_AFFINITY=1 -DENABLE_DSR_ICMP_ERRORS=1 -DENABLE_DSR=1 -DENABLE_DSR_HYBRID=1 -DENABLE_IPV4_FRAGMENTS=1 -DENABLE_EGRESS_GATEWAY=1 -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1 -I/root/cyclinder/cilium/bpf/include -I/root/cyclinder/cilium/bpf -D__NR_CPUS__=8 -O2 -g -target bpf -std=gnu89 -nostdinc -emit-llvm -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -c bpf_lxc.c -o bpf_lxc.ll
error: unknown target triple 'bpf', please use -triple or -arch
make[2]: *** [bpf_lxc.ll] 错误 1
make[2]: 离开目录“/root/cyclinder/cilium/bpf”
make[1]: *** [bpf] 错误 2
make[1]: 离开目录“/root/cyclinder/cilium”
make: *** [cilium-build] 错误 2
make: 离开目录“/root/cyclinder/cilium/Documentation”
[root@localhost cilium]# bpf^C
[root@localhost cilium]# ^C
[root@localhost cilium]# uname -a
Linux localhost.localdomain 5.4.228-1.el7.elrepo.x86_64 #1 SMP Thu Dec 15 18:03:40 EST 2022 x86_64 x86_64 x86_64 GNU/Linux
```

I don't want to spend any more time on this 😅, is there a way to replace this command?

--- a/Documentation/cmdref/cilium-agent.md
+++ b/Documentation/cmdref/cilium-agent.md
@@ -78,6 +78,7 @@ cilium-agent [flags]
       --dns-policy-unload-on-shutdown                           Unload DNS policy rules on graceful shutdown
       --dnsproxy-concurrency-limit int                          Limit concurrency of DNS message processing
       --dnsproxy-concurrency-processing-grace-period duration   Grace time to wait when DNS proxy concurrent limit has been reached during DNS message processing
+      --egress-gateway-policy-map-max int                       Maximum number of entries in egress gateway policy map (default 16384)
       --egress-masquerade-interfaces string                     Limit egress masquerading to interface selector
       --egress-multi-home-ip-rule-compat                        Offset routing table IDs under ENI IPAM mode to avoid collisions with reserved table IDs. If false, the offset is performed (new scheme), otherwise, the old scheme stays in-place.
       --enable-auto-protect-node-port-range                     Append NodePort range to net.ipv4.ip_local_reserved_ports if it overlaps with ephemeral port range (net.ipv4.ip_local_port_range) (default true)

--- a/daemon/cmd/daemon_main.go
+++ b/daemon/cmd/daemon_main.go
@@ -64,6 +64,7 @@ import (
 	"github.com/cilium/cilium/pkg/maglev"
 	"github.com/cilium/cilium/pkg/maps/ctmap"
 	"github.com/cilium/cilium/pkg/maps/ctmap/gc"
+	"github.com/cilium/cilium/pkg/maps/egressmap"
 	"github.com/cilium/cilium/pkg/maps/lbmap"
 	"github.com/cilium/cilium/pkg/maps/nat"
 	"github.com/cilium/cilium/pkg/maps/neighborsmap"
@@ -666,6 +667,9 @@ func initializeFlags() {
 
 	flags.Bool(option.InstallEgressGatewayRoutes, false, "Install egress gateway IP rules and routes in order to properly steer egress gateway traffic to the correct ENI interface")
 	option.BindEnv(Vp, option.InstallEgressGatewayRoutes)
+
+	flags.Int(option.EgressGatewayPolicyMapEntriesName, egressmap.MaxPolicyEntries, "Maximum number of entries in egress gateway policy map")
+	option.BindEnv(Vp, option.EgressGatewayPolicyMapEntriesName)
 
 	flags.Bool(option.EnableEnvoyConfig, false, "Enable Envoy Config CRDs")
 	option.BindEnv(Vp, option.EnableEnvoyConfig)

--- a/daemon/cmd/datapath.go
+++ b/daemon/cmd/datapath.go
@@ -345,7 +345,7 @@ func (d *Daemon) initMaps() error {
 	}
 
 	if option.Config.EnableIPv4EgressGateway {
-		if err := egressmap.InitEgressMaps(); err != nil {
+		if err := egressmap.InitEgressMaps(option.Config.EgressGatewayPolicyMapEntries); err != nil {
 			return err
 		}
 	}

--- a/install/kubernetes/cilium/templates/cilium-configmap.yaml
+++ b/install/kubernetes/cilium/templates/cilium-configmap.yaml
@@ -834,6 +834,9 @@ data:
 {{- if .Values.egressGateway.installRoutes }}
   install-egress-gateway-routes: "true"
 {{- end }}
+{{- if .Values.egressGateway.maxPolicyEntries }}
+  egress-gateway-policy-map-max: {{ .Values.egressGateway.maxPolicyEntries }}
+{{- end }}
 
 {{- if hasKey .Values "vtep" }}
   enable-vtep: {{ .Values.vtep.enabled | quote }}

--- a/install/kubernetes/cilium/values.yaml
+++ b/install/kubernetes/cilium/values.yaml
@@ -1502,6 +1502,8 @@ egressGateway:
   # -- Install egress gateway IP rules and routes in order to properly steer
   # egress gateway traffic to the correct ENI interface
   installRoutes: false
+  # -- Maximum number of entries in egress gateway policy map
+  # maxPolicyEntries: 16384
 
 vtep:
 # -- Enables VXLAN Tunnel Endpoint (VTEP) Integration (beta) to allow

--- a/install/kubernetes/cilium/values.yaml.tmpl
+++ b/install/kubernetes/cilium/values.yaml.tmpl
@@ -1499,6 +1499,8 @@ egressGateway:
   # -- Install egress gateway IP rules and routes in order to properly steer
   # egress gateway traffic to the correct ENI interface
   installRoutes: false
+  # -- Maximum number of entries in egress gateway policy map
+  # maxPolicyEntries: 16384
 
 vtep:
 # -- Enables VXLAN Tunnel Endpoint (VTEP) Integration (beta) to allow

--- a/pkg/datapath/linux/config/config.go
+++ b/pkg/datapath/linux/config/config.go
@@ -150,7 +150,7 @@ func (h *HeaderfileWriter) WriteNodeConfig(w io.Writer, cfg *datapath.LocalNodeC
 	cDefinesMap["IPCACHE_MAP"] = ipcachemap.Name
 	cDefinesMap["IPCACHE_MAP_SIZE"] = fmt.Sprintf("%d", ipcachemap.MaxEntries)
 	cDefinesMap["EGRESS_POLICY_MAP"] = egressmap.PolicyMapName
-	cDefinesMap["EGRESS_POLICY_MAP_SIZE"] = fmt.Sprintf("%d", egressmap.MaxPolicyEntries)
+	cDefinesMap["EGRESS_POLICY_MAP_SIZE"] = fmt.Sprintf("%d", option.Config.EgressGatewayPolicyMapEntries)
 	cDefinesMap["SRV6_VRF_MAP4"] = srv6map.VRFMapName4
 	cDefinesMap["SRV6_VRF_MAP6"] = srv6map.VRFMapName6
 	cDefinesMap["SRV6_POLICY_MAP4"] = srv6map.PolicyMapName4

--- a/pkg/egressgateway/manager_privileged_test.go
+++ b/pkg/egressgateway/manager_privileged_test.go
@@ -120,7 +120,7 @@ func (k *EgressGatewayTestSuite) SetUpSuite(c *C) {
 	option.Config.EnableIPv4EgressGateway = true
 	option.Config.InstallEgressGatewayRoutes = true
 
-	egressmap.InitEgressMaps()
+	egressmap.InitEgressMaps(egressmap.MaxPolicyEntries)
 
 	nodeTypes.SetName(node1)
 }

--- a/pkg/maps/egressmap/egress.go
+++ b/pkg/maps/egressmap/egress.go
@@ -14,11 +14,11 @@ var (
 )
 
 // InitEgressMaps initializes the egress policy map.
-func InitEgressMaps() error {
-	return initEgressPolicyMap(PolicyMapName, true)
+func InitEgressMaps(maxPolicyEntries int) error {
+	return initEgressPolicyMap(PolicyMapName, maxPolicyEntries, true)
 }
 
 // OpenEgressMaps initializes the egress policy map.
 func OpenEgressMaps() error {
-	return initEgressPolicyMap(PolicyMapName, false)
+	return initEgressPolicyMap(PolicyMapName, 0, false)
 }

--- a/pkg/maps/egressmap/egress_privileged_test.go
+++ b/pkg/maps/egressmap/egress_privileged_test.go
@@ -35,7 +35,7 @@ func (k *EgressMapTestSuite) SetUpSuite(c *C) {
 }
 
 func (k *EgressMapTestSuite) TestEgressMap(c *C) {
-	err := initEgressPolicyMap(PolicyMapName, true)
+	err := initEgressPolicyMap(PolicyMapName, MaxPolicyEntries, true)
 	c.Assert(err, IsNil)
 	defer EgressPolicyMap.Unpin()
 

--- a/pkg/maps/egressmap/policy.go
+++ b/pkg/maps/egressmap/policy.go
@@ -39,7 +39,7 @@ type egressPolicyMap struct {
 }
 
 // initEgressPolicyMap initializes the egress policy map.
-func initEgressPolicyMap(policyMapName string, create bool) error {
+func initEgressPolicyMap(policyMapName string, maxPolicyEntries int, create bool) error {
 	var m *ebpf.Map
 
 	if create {
@@ -48,7 +48,7 @@ func initEgressPolicyMap(policyMapName string, create bool) error {
 			Type:       ebpf.LPMTrie,
 			KeySize:    uint32(unsafe.Sizeof(EgressPolicyKey4{})),
 			ValueSize:  uint32(unsafe.Sizeof(EgressPolicyVal4{})),
-			MaxEntries: uint32(MaxPolicyEntries),
+			MaxEntries: uint32(maxPolicyEntries),
 			Pinning:    ebpf.PinByName,
 		})
 

--- a/pkg/option/config.go
+++ b/pkg/option/config.go
@@ -636,6 +636,10 @@ const (
 	// entries.
 	SockRevNatEntriesName = "bpf-sock-rev-map-max"
 
+	// EgressGatewayPolicyMapEntriesName configures max entries for egress gateway's policy
+	// map.
+	EgressGatewayPolicyMapEntriesName = "egress-gateway-policy-map-max"
+
 	// LogSystemLoadConfigName is the name of the option to enable system
 	// load loggging
 	LogSystemLoadConfigName = "log-system-load"
@@ -1477,6 +1481,10 @@ type DaemonConfig struct {
 	// SockRevNatEntries is the maximum number of sock rev nat mappings
 	// allowed in the BPF rev nat table
 	SockRevNatEntries int
+
+	// EgressGatewayPolicyMapEntries is the maximum number of entries
+	// allowed in the BPF egress gateway policy map.
+	EgressGatewayPolicyMapEntries int
 
 	// DisableCiliumEndpointCRD disables the use of CiliumEndpoint CRD
 	DisableCiliumEndpointCRD bool
@@ -2956,6 +2964,7 @@ func (c *DaemonConfig) Populate(vp *viper.Viper) {
 	c.EnableIPMasqAgent = vp.GetBool(EnableIPMasqAgent)
 	c.EnableIPv4EgressGateway = vp.GetBool(EnableIPv4EgressGateway)
 	c.InstallEgressGatewayRoutes = vp.GetBool(InstallEgressGatewayRoutes)
+	c.EgressGatewayPolicyMapEntries = vp.GetInt(EgressGatewayPolicyMapEntriesName)
 	c.EnableEnvoyConfig = vp.GetBool(EnableEnvoyConfig)
 	c.EnableIngressController = vp.GetBool(EnableIngressController)
 	c.EnableGatewayAPI = vp.GetBool(EnableGatewayAPI)

